src/Compat/CompatController.php

```diff
@@ -420,6 +421,69 @@ public function sendExtraUpdates(array $updates)
         $this->getResponse()->setHeader('X-Icinga-Extra-Updates', join(',', $extraUpdates));
     }
 
+    /**
+     * Close the modal content and refresh the related view
+     *
+     * NOTE: When you use this with older Icinga Web versions (< 2.12), you will need to specify a valid redirect url,
+     * that will produce the same result as using the `__REFRESH__` redirect with the latest Icinga Web version.
+     *
+     * This is supposed to be used in combination with a modal view and closes only the modal content,
+     * and refreshes the modal opener (regardless of whether it is col1 or col2).
+     *
+     * @param null|Url|string $url
+     * @param array<int|string, string> $extraUpdates Provide containers other than the redirect target to be refreshed
+     *        after the redirection. Is just for compatibility reasons and won't be used with latest Icinga Web
+     *        versions. See {@see static::sendExtraUpdates()} how to properly provide extra updates.
```

Comment:
Thinking about this, I don't see the need for this being an array. A bool should suffice which allows the caller to define that also `col1` should be refreshed.

If the modal opener is col1, the url refreshes it. If the opener is col2, only col1 might be related. Especially since there's no col3 yet.